app/routes/titiler/algorithms/integrated_alerts.py

```diff
@@ -0,0 +1,30 @@
+from collections import OrderedDict, namedtuple
+from .alerts import Alerts
+
+from app.models.enumerators.alerts_confidence import DeforestationAlertConfidence
+
+
+class IntegratedAlerts(Alerts):
+    title: str = "Integrated Deforestation Alerts"
+    description: str = "Decode and visualize deforestation alerts"
+
+    Colors: namedtuple = namedtuple("Colors", ["red", "green", "blue"])
```

Comment:
Same here, any reason to include Colors here if it's the same as the base class?